app/javascript/components/get-weather-query.jsx

```diff
@@ -43,49 +43,56 @@ export const WeatherQuery = ({
   WeatherFrag,
   queryName,
   cityName,
-}) => (
-  <Query
-    query={getLocationWeather(WeatherFrag, queryName)}
-    variables={{ cityName }}
-  >
-    {({ loading, error, data, subscribeToMore }) => {
-      if (loading) {
-        return <LoadingMessage />;
-      }
-      if (error) {
+  resetTimer,
```

Comment:
Maybe a better name? `resetTimer` had me thinking we reset the timer back to 0 when we're completed/errored but I'm guessing this call starts the transition timer.

Reply:
Maybe back to `startTimer`, since that's the action that this function is dispatching?